client/src/style/scss/overrides.scss

```diff
@@ -88,7 +88,7 @@ button {
 // Modal -- wider by default, scroll like Trello
 
 .modal-dialog {
-    width: 690px;
+    // width: 690px; // by removing this, size can be set with the "size" attribute of the bootstrap component <b-modal>
```

Comment:
Someone set a fixed the size of all modals globally. I am not sure if we are allowed to remove this, but since we have so much text in a modal it sure is nice to be able to make it a litle wider :smile:

Reply:
Maybe you can override this in your modal directly using "scoped" styles.

Reply:
I was not able to. But I restored the original state anyways in order to not introduce any accidental changes.
The modal is a bit thin but still looks OK :+1: